app/helpers/time_helper.rb

```diff
@@ -39,7 +39,7 @@ def formatted_datetime(datetime, year: false)
     datetime.strftime(format)
   end
 
-  def formatted_business_days_ago(time)
+  def business_days_ago(time)
```

Comment:
this name change is a nitpick - the method just returns the number of days, which to me doesn't seem "formatted"